app/src/main/res/values/strings.xml

```diff
@@ -48,6 +48,8 @@
     <string name="pref_code_group_size_title">Code digit grouping</string>
     <string name="pref_code_group_size_summary">Select number of digits to group codes by</string>
     <string name="pref_account_name_position_title">Show the account name</string>
+    <string name="pref_expiration_state_title">Show when codes are about to expire</string>
+    <string name="pref_expiration_state_summary">Change the color of the code and have the codes blink when they are about to expire</string>
```

Comment:
We should probably pick either "code" or "codes". Or alternatively: "Change the color of the codes and have them blink when they are about to expire"